src/components/repository/CreateRepositoryForm.vue

```diff
@@ -224,13 +227,13 @@ export default {
 
     .create-repository {
         display: flex;
-        flex-wrap: wrap;
-        justify-content: center;
+        justify-content: space-around;
```

Comment:
Não fica muito espaçado o form e o card quando a tela é grande?

Reply:
Realmente fica mais espaçado, mas, vai evitar ficar bugado quando a tela for pequena 
![image](https://user-images.githubusercontent.com/61828136/87308218-4579e400-c4f1-11ea-903e-d80cd4198e30.png)